src/dreammaker/builtins.rs

```diff
@@ -376,7 +394,7 @@ pub fn register_builtins(tree: &mut ObjectTree) {
             alpha, color, infra_luminosity, layer, maptext_width, maptext_height,
             maptext_x, maptext_y, luminosity, pixel_x, pixel_y, pixel_w, pixel_z,
             transform, dir, icon, icon_state, invisibility, maptext, suffix, appearance,
-            dir, radius,
+            dir, radius, space,
```

Comment:
note: if `space` is specified, `color` must be as well - potential linting/testing in the future?

Reply:
possible feature for the in-depth `filter()` verifier